src/main/java/com/joss/conductor/mobile/ConductorConfig.java

```diff
@@ -474,4 +474,12 @@ public SauceOnDemandAuthentication getSauceAuthentication(String sauceUserName,
     public boolean isLocal() {
         return getHub() == null;
     }
+    public boolean isHubLocal() {
+        URL url = getHub();
+        String sHub = "";
+        if (!(url == null)){
+            sHub = url.toString();
+        }
+        return !sHub.contains("saucelabs");
```

Comment:
you can probably one-line this whole thing, something like:
`return !isLocal() && !getHub().toString().contains("saucelabs");`

Also, I don't know if I would suggest hardcoding comparing to "saucelabs" because they could simply change the domain name and this will stop working.

I would suggest changing this to maybe a boolean flag that you can set to use either `localhost` or some other thing we want.

Reply:
We will modify the script to single line return.
And for saucelab connection - we will modify accordingly. add the flag isLocalHub from yaml and try implementing.
Thank you.

Reply:
and to be clear the credit for this goes to Praneeth.